src/en/charms-deploying.md

```diff
@@ -1,67 +1,89 @@
 Title: Deploying services  
+TODO: First section spent defining charms. This should all be placed in charms.md and
+	linked to from here. As of this writing, that page does not cover local charms.
+      Removed (commented out) section 'References and examples' due to the
+	references being 3 years old and not offering much at all.
+      This page is too long and should be broken up (or apply the fabled TOC).
+      Review: 'channel support' points (especially #2). See https://goo.gl/IKzRsD .
 
-# Deploying Services
+
+# Deploying services
 
 The fundamental point of Juju is that you can use it to deploy services through
 the use of charms (the magic bits of code that make things just work). These
-charms can be fetched from the charm store, stored in a local repository, or if
-you are feeling clever, written by you. Just as there are different series of
-Ubuntu ('precise', 'raring', etc), so there needs to be different series of
-charms to take into account any subtle changes in the underlying OS. For the
-most part you can forget about this, as Juju will always try to apply the most
-relevant charm, so deploying can be straightforward and easy.
+charms can exist in the [Charm Store](https://jujucharms.com/store) or on the
+file system (previously downloaded from the store or written locally).
+
+Charms use the concept of *series* analogous as to how Juju does with Ubuntu
```

Comment:
Yeah. I think we need to put across the concepts that:

* The charm may be written differently depending on what OS it is deployed to. This is especially true of charms targetting Xenial for example, which have to use systemd instead of upstart.

* You can choose which charm you want to use, and you can deploy a charm on a series that it doesn't claim to support if you really want}}

* ... however, to have things 'just work', don't bother specifying anyhting and the charm will decide which OS/version to use without any additional hassle